mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ignition
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	ignv3types "github.com/coreos/ignition/v2/config/v3_0/types"
+	"github.com/pkg/errors"
+
+	"github.com/coreos/mantle/kola"
+	"github.com/coreos/mantle/kola/cluster"
+	"github.com/coreos/mantle/kola/register"
+	"github.com/coreos/mantle/platform"
+	"github.com/coreos/mantle/util"
+)
+
+func init() {
+	register.RegisterTest(&register.Test{
+		Name:        "coreos.ignition.failure",
+		Run:         runIgnitionFailure,
+		ClusterSize: 0,
+		Platforms:   []string{"qemu-unpriv"},
+	})
+}
+
+func runIgnitionFailure(c cluster.TestCluster) {
+	if err := ignitionFailure(c); err != nil {
+		c.Fatal(err.Error())
+	}
+}
+
+func ignitionFailure(c cluster.TestCluster) error {
```

Comment:
Hmm, I don't follow this test. Isn't this... starting a VM inside the test VM?

I guess this is testing that the log capture works. Though maybe we can just break an existing test instead and check that CI fails and dumps the full journal before unbreaking it for merge?

Reply:
Note `ClusterSize: 0` - that's a way to write arbitrary code as part of a kola test.

Reply:
> Though maybe we can just break an existing test instead and check that CI fails and dumps the full journal before unbreaking it for merge?

I've already done that a lot =)  But I think we should test failure cases too.

Reply:
Ahh right, missed that!